src/components/Auth/LoginForm/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, {useEffect} from 'react';
+import {useTranslation} from "react-i18next";
+import {useForm} from "react-hook-form";
+import {yupResolver} from "@hookform/resolvers/yup";
+import { useGoogleLogin } from '@react-oauth/google';
+import {IconButton, InputAdornment } from '@mui/material';
+import Input from '@components/Input';
+import {
+  AuthContainer,
+  AuthForm,
+  AuthGoogleContainer,
+  AuthInput,
+  AuthInputTitle,
+  AuthLinkContainer,
+  AuthLinkToLogin,
+  AuthSendButton,
+  AuthText,
+  AuthTitle,
+  Form, GoogleImg,
+  GoogleText,
+  PasswordImg
+} from '@components/Auth/styles';
+import { ILogin } from '@components/Auth/type';
+import visible from "@assets/auth/eye.svg";
+import visibleOff from "@assets/auth/eyeSlash.svg";
+import google from '@assets/auth/google.svg'
+import { email, end, password } from '@constants/auth';
+import { LoginSchema } from '@validation/auth.validate';
+import { PATH } from '@router/index';
+import { NavLink } from 'react-router-dom';
+
+function LoginForm() {
+  const [showPassword, setShowPassword] = React.useState(false);
+
+  const handleClickShowPassword = () => setShowPassword((show) => !show);
+
+  const { t } = useTranslation();
+  const {
+    register,
+    handleSubmit,
+    control,
+    formState: { errors, isValid }
+  } = useForm<ILogin>({
+    mode: 'onChange',
+    defaultValues: {
+      email: '',
+      password: ''
+    },
+    resolver: yupResolver(LoginSchema),
+  });
+  useEffect(() => {
+    register('password');
+  }, []);
+
+  const onSubmit = (data: ILogin) => { };
+
+  const login = useGoogleLogin({ });
```

Comment:
you have mess, states should be with states, hooks with hooks, handlers with handlers